### PR TITLE
Added Dutch (nl) localization

### PR DESCRIPTION
**Pull request by yasingunaydiin**
I'm still working on it but I'll do my best to finish it as soon as possible.

### CODE REVIEW

**Comment by 1024jp** [comment]
Thank you for your PR.
I confirmed that the changes seem correctly pushed this time.
You have so far done 6 .xcstrings files, correct?

Keep on and let me know if all finished.

**Comment by yasingunaydiin** [comment]
Correct!

Now i can work in peace hah. Thank you for your patience!

[image: user img]
Yasin Günaydın
Localization Specialist/ Interpreter Dutch & Turkish

   - [image: Mobile icon] Mobile: +905417688751


   - [image: linkedIn icon]
<https://www.linkedin.com/in/yasin-gunaydin-b45466204/>



Op zo 7 apr 2024 om 16:24 schreef 1024jp ***@***.***>

**Comment by 1024jp** [comment]
@yasingunaydiin Hi, this is a reminder in case you may not have noticed that there are some localizable files also in the Storyboard section. Toggle the chevrons of each storyboard and you'll find them.

Screenshot 2024-04-12 at 23 30 17

Meanwhile, I'll review your work in the Localizable section.

**Review by 1024jp — CHANGES_REQUESTED** [review]
I've reviewed all the localized strings until this point and found some potential strings that could be improved. I'll suggest them later when all the strings are once localized.

But at this point, I would like to share the following two points with you:
- Please consider updating the string I mentioned.
- The pair of double quotation marks in English seems to be localized to straight quotation marks in Dutch in macOS. For instance, `“%@”` will be `'%@'` in Dutch. (For those that have already been localized, I can convert them by myself. Just please be careful about what you are going to localize.)

**Comment by 1024jp** [comment]
> Yes im currently working on them and am almost done with them.

Thank you! I realized I was a bit impatient.

File: CotEditor/mul.lproj/Document.xcstrings [hunk 1]
```diff
@@ -3657,6 +3963,12 @@
             "value" : "選択範囲をコメントアウト/コメント解除"
           }
         },
+        "nl" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Selectie selecteren of verwijderen  "
```
[the remark this answers is not in the document]
Comment (yasingunaydiin) [reply CotEditor/mul.lproj/Document.xcstrings:3969]: Will look into it. I remember I had a hard time translating this so I looked at the German and Turkish translation for hints.

**Review by 1024jp — APPROVED** [review]

**Comment by 1024jp** [comment]
Thank you! Perfect!

I've merged the changes and released CotEditor 4.8.0-rc so that you can check the actual implementation easily.
Please check for any oddities.
https://github.com/coteditor/CotEditor/releases/tag/4.8.0-rc

In addition, I have made some improvements on my end to use macOS-specific expressions.
I don't think there are grammatical problems, but it would be helpful if you could confirm also this change just in case.
https://github.com/coteditor/CotEditor/commit/692b764056053ce540bc5a564201095e60635360

Once all issues in this 4.8.0-rc are fixed, I will release the official version by the end of this month.

**Pull request merged**